anime_downloader/extractors/kwik.py

```diff
@@ -15,26 +15,23 @@ class Kwik(BaseExtractor):
 
     def _get_data(self):
 
-        # Need a javascript deobsufication api/python, so someone smarter
-        # than me can work on that for now I will add the pattern I observed
-
-        # alternatively you can pattern match on `src` for stream_url part
-        source_parts_re = re.compile(r'action=\"([^"]+)\".*value=\"([^"]+)\".*Click Here to Download',
-                                     re.DOTALL)
-
         # Kwik servers don't have direct link access you need to be referred
         # from somewhere, I will just use the url itself.
 
         download_url = self.url.replace('kwik.cx/e/', 'kwik.cx/f/')
-
         kwik_text = helpers.get(download_url, referer=download_url).text
-        post_url, token = source_parts_re.search(kwik_text).group(1, 2)
+        deobfuscated_text = (jsbeautifier.beautify(kwik_text))
+
+        regex_all = (r"var _[\W\w]{5}=\"[\W\w]*?\"")
```

Comment:
regex= (r"var _[\W\w]{5}=\"([\W\w]*?)\"")
token = re.search(regex,deobfuscated_text).group(1)[::-1]

Reply:
Smart!
Fixed it